### PR TITLE
Go to top button

### PR DESCRIPTION
**Pull request by Primdahl26**
### Is your feature request related to a problem? Please describe.

It can get annoying when getting far down in scrolling, and having to go back to the top.

### Describe the solution you'd like

We need a button or something similar that can solve that

### CODE REVIEW

**Comment by Primdahl26** [comment]
The button only appears when scrolled down.

image
image

File: frontend/src/components/back_to_top_button.svelte [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  }
+
+  function scrollContainer() {
+    return document.documentElement || document.body
+  }
+
+  function handleOnScroll() {
+    if (!scrollContainer()) {
+      return
+    }
```
Comment (AndreasPB) [inline frontend/src/components/back_to_top_button.svelte:16]: WTB es6 syntax

Comment (Primdahl26) [reply frontend/src/components/back_to_top_button.svelte]: Done

**Review by AndreasPB — APPROVED** [review]

**Pull request merged**